java-manta-cli/src/main/java/com/joyent/manta/cli/MantaCLI.java

```diff
@@ -17,364 +19,367 @@
 import com.joyent.manta.config.EnvVarConfigContext;
 import com.joyent.manta.config.MapConfigContext;
 import com.joyent.manta.util.MantaVersion;
-import org.apache.commons.lang3.ArrayUtils;
-import org.apache.commons.lang3.StringUtils;
-import org.apache.commons.lang3.exception.ExceptionUtils;
+import org.slf4j.LoggerFactory;
+import picocli.CommandLine;
 
 import java.io.File;
 import java.io.IOException;
 import java.io.UncheckedIOException;
 import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.security.NoSuchAlgorithmException;
+import java.util.List;
 import java.util.stream.Stream;
 import javax.crypto.SecretKey;
 import javax.crypto.spec.SecretKeySpec;
 
-/**
- * Class providing a CLI interface to the Java Manta SDK.
- *
- * @author <a href="https://github.com/dekobon">Elijah Zupancic</a>
- * @since 3.0.0
- */
+@CommandLine.Command(name = "java-manta-cli", sortOptions = false,
```

Comment:
"Picocli will only emit ANSI escape codes on supported platforms."

If I pipe to a file all  looks well (no-colors or malformed escape codes).